examples/opf/tools/sp_plotter.py

```diff
@@ -330,17 +330,16 @@ def testSPNew():
   
   
   # Setup a SP
-  sp = FDRCSpatial2.FDRCSpatial2(
```

Comment:
> @david-ragazzi I think if you create the ~/Desktop/ExperimentResults directory, you might be able to run the examples. 

It didn't work. Actually `Basil100x21.csv` already must exists because it is a input data file. I didn't find this file in anywhere! Is someone using this example? It seems a bit forgoten in repo.